VERSIONING_POLICY.md

```diff
@@ -0,0 +1,33 @@
+# Open OnDemand Versioning Policy
+## Version Numbering
+Open OnDemand code releases utilize a XXX.YYY.ZZZ numbering scheme (e.g. 2.0.29), where:
+* XXX is the major version number
+* YYY is the minor version number
+* ZZZ is the patch version number
+
+## Major Versions
+New major version numbers will typically be implemented for any of the following reasons:
+* Significant breaking changes that prevent seamless upgrading from previous versions
+* Significant deprecations related to software dependencies
+* Large sets of new features or functionalities
+
+## Minor Versions
+New minor version numbers will typically be implemented for any of the following reasons:
+* New features of functionalities
+* Large sets of bug fixes
+
+## Patch Versions
+New patch version numbers will typically be implemented for any of the following reasons:
+* Security updates
+* Minor bug fixes
+
+## Tagging
+Intermediate or transient releases will be indicated via additional tagging, including:
+* Pre-release, for releases that are being actively tested in production at early adopter sites
```

Comment:
This seems a little confusion.  Does the RC# represent the Pre-release?

Reply:
Per-chatGPT:

A pre-release is a version of the software that is not yet considered stable or ready for production use. It is typically released for testing, gathering feedback, and identifying bugs. Pre-release versions come after the PATCH version number and are denoted by a hyphen followed by an identifier, e.g., 1.0.0-alpha, 1.0.0-beta, or 1.0.0-beta.1. The identifiers are usually ordered alphabetically, with "alpha" typically denoting an early-stage pre-release and "beta" denoting a more mature but not yet final pre-release.

A release candidate (RC) is a more mature pre-release version that is considered feature-complete and has passed thorough testing. It is believed to be stable enough for production use but may still have some undiscovered issues. A release candidate is often the final step before declaring the software as stable and ready for general availability (GA). In SemVer, release candidates are denoted with an "rc" identifier, e.g., 1.0.0-rc.1, `1.0.0-rc.2`.

Reply:
Feel free to suggest some alternative wording here.  I don't have a strong opinion on this.  I just looked at the release history and noticed we have certain things 'tagged' pre-release and other's tagged things like .rc1, .rc2, etc.  Note I'm using the phrase 'tagging' generically here, because I think the pre-release and .rc1 text I saw are technically different things in GitHub speak?

Reply:
I updated this to just say release and release candidate.  IDK if the pre-release lingo is beneficial, especially given the conversation/questions here.